### ISSUE TITLE
Not enough attributes defined for PDC::V1::Product

### ISSUE THREAD

**Issue by simonbaird** [product-definition-center/pdc-ruby-gem#22]
In rails console:

````
[5] pry(main)> require 'pdc_config'
false
[6] pry(main)> PDC::V1::Product.find('ceph')
Setting unknown attribute: name PDC::V1::Product
Setting unknown attribute: short PDC::V1::Product
Setting unknown attribute: active PDC::V1::Product
Setting unknown attribute: product_versions PDC::V1::Product
Setting unknown attribute: internal PDC::V1::Product
#<PDC::V1::Product(v1/products/(:product_id)) id: nil name: "Red Hat Ceph Storage" short: "ceph" active: true product_versions: ["ceph-1", "ceph-2"] internal: false>

[7] pry(main)> PDC::V1::Product.attributes
[
    [0] "product_id"
]
[8] pry(main)> PDC::V1::Release.attributes
[
    [ 0] "release_id",
    [ 1] "short",
    [ 2] "version",
    [ 3] "name",
    [ 4] "base_product",
    [ 5] "active",
    [ 6] "product_version",
    [ 7] "release_type",
    [ 8] "compose_set",
    [ 9] "integrated_with",
    [10] "bugzilla",
    [11] "dist_git",
    [12] "brew",
    [13] "product_pages",
    [14] "errata"
]
````

It still works, but I get the "Setting unknown attribute:" warnings. It would be nice if product had at least these attributes defined:

name
short
active
product_versions
internal

**Comment by simonbaird** [comment]
PDC::V1::Product.all spits out hundreds of these warnings and they're appearing in the Jenkins console output, so it is quite annoying, even though not too urgent.

**Comment by sthaha** [comment]
Fixed in commit - f325787
